phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1997,26 +1997,35 @@ private PTable createTableInternal(CreateTableStatement statement, byte[][] spli
             Long viewTTL = VIEW_TTL_NOT_DEFINED;
             Long viewTTLHighWaterMark = MIN_VIEW_TTL_HWM;
             Long viewTTLProp = (Long) TableProperty.VIEW_TTL.getValue(tableProps);;
+            // Validate VIEW_TTL prop value if set
+            if (viewTTLProp != null) {
+                if (viewTTLProp < 0) {
```

Comment:
@jpisaac I guess there is no upper-bound?

Reply:
Wait, upper bound is `HConstants.LATEST_TIMESTAMP`. Is it possible to assign a larger value than that from the client?

Reply:
HConstants.LATEST_TIMESTAMP = Long.MAX_VALUE